ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +340,44 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def export_pgoutput_to_googlesheet(self, pg_output, sheet_index):
+        """
+        Collect pgbench output to google spreadsheet
+
+        Args:
+            pg_output (list):  pgbench outputs in list
+            sheet_name (str): Name of the sheet
+            sheet_index (int): Index of sheet
+
+        """
+        # Collect data and export to Google doc spreadsheet
+        g_sheet = GoogleSpreadSheetAPI(
+            sheet_name='E2E Workloads - PGSQL', sheet_index=sheet_index
+        )
+        log.info("Exporting pgoutput data to google spreadsheet")
+        for i in range(len(pg_output)):
```

Comment:
I suggest following:
1) All details like versions of OCP, OCS, ceph etc., to be shown in table format
2) have pgbench pod name in first column and rest of the columns like lat_avg etc., from 2nd column

Reply:
Resolved in the new commit